src/map/lua/lua_baseentity.cpp

```diff
@@ -6765,9 +6764,8 @@ void CLuaBaseEntity::addQuest(uint8 questLogID, uint16 questID)
     if (questLogID < MAX_QUESTAREA && questID < MAX_QUESTID)
     {
         uint8 current  = PChar->m_questLog[questLogID].current[questID / 8] & (1 << (questID % 8));
-        uint8 complete = PChar->m_questLog[questLogID].complete[questID / 8] & (1 << (questID % 8));
 
-        if ((current == 0) && (complete == 0))
```

Comment:
not sure when that came in but that has to have been a mistake, it was never supposed to block based on complete state and I know somewhere in the past I was able to have active+complete like on retail